presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -126,6 +126,23 @@ public void testSTCentroid()
         assertFunction("ST_AsText(ST_Centroid(ST_GeometryFromText('POLYGON ((0 0, 0 5, 5 5, 5 0, 0 0), (1 1, 1 2, 2 2, 2 1, 1 1))')))", VARCHAR, "POINT (2.5416666666666665 2.5416666666666665)");
     }
 
+    @Test
+    public void testSTConvexNull()
+    {
+        // test Invalid types
+        assertInvalidFunction("ST_AsText(ST_ConvexNull(ST_GeometryFromText('LINESTRING EMPTY')))", VARCHAR, "ST_Polygon only applies to MULTI_LINE_STRING or MULTI_POINT or MULTI_POLYGON or GEOMETRY_COLLECTION\n. Input type is: LINE_STRING");
+        assertInvalidFunction("ST_AsText(ST_ConvexNull(ST_GeometryFromText('POINT (3 5)')))", VARCHAR, "ST_Polygon only applies to MULTI_LINE_STRING or MULTI_POINT or MULTI_POLYGON or GEOMETRY_COLLECTION\n. Input type is: POINT");
+        assertInvalidFunction("ST_AsText(ST_ConvexNull(ST_GeometryFromText('LINESTRING (1 1, 2 2, 3 3)')))", VARCHAR, "ST_Polygon only applies to MULTI_LINE_STRING or MULTI_POINT or MULTI_POLYGON or GEOMETRY_COLLECTION\n. Input type is: LINESTRING");
+        assertInvalidFunction("ST_AsText(ST_ConvexNull(ST_GeometryFromText('POLYGON ((1 1, 1 4, 4 4, 4 1)')))", VARCHAR, "ST_Polygon only applies to MULTI_LINE_STRING or MULTI_POINT or MULTI_POLYGON or GEOMETRY_COLLECTION\n. Input type is: POLYGON");
+
+        // test valid types
+        assertFunction("ST_AsText(ST_ConvexNull(ST_GeometryFromText('MULTILINESTRING EMPTY')))", VARCHAR, "POLYGON EMPTY");
```

Comment:
Could you add tests for all types of empty geometries? E.g. POINT EMPTY, MULTIPOINT EMPTY, LINESTRING EMPTY, MULTILINESTRING EMPTY, POLYGON EMPTY, MULTIPOLYGON EMPTY, GEOMETRYCOLLECTION EMPTY.